docs/PACKAGES.md

```diff
@@ -105,7 +105,7 @@ The dependencies between layers are enforced by the layer-check command._
+| - [@fluid-internal/client-api](/packages/runtime/client-api)</br>- [fluid-framework](/experimental/framework/fluid-framework)</br>- [@fluid-experimental/data-objects](/experimental/framework/data-objects)</br>- [@fluid-experimental/property-binder](/experimental/PropertyDDS/packages/property-binder)</br>- [@fluid-experimental/property-changeset](/experimental/PropertyDDS/packages/property-changeset)</br>- [@fluid-experimental/property-common](/experimental/PropertyDDS/packages/property-common)</br>- [@fluid-experimental/property-dds](/experimental/PropertyDDS/packages/property-dds)</br>- [@fluid-experimental/property-inspector-table](/experimental/PropertyDDS/packages/property-inspector-table)</br>- [@fluid-experimental/property-properties](/experimental/PropertyDDS/packages/property-properties)</br>- [@fluid-experimental/property-proxy](/experimental/PropertyDDS/packages/property-proxy)</br>- [@fluid-experimental/last-edited](/experimental/framework/last-edited)</br>- [@fluidframework/aqueduct](/packages/framework/aqueduct)</br>- [@fluidframework/data-object-base](/packages/framework/data-object-base)</br>- [@fluidframework/dds-interceptions](/packages/framework/dds-interceptions)</br>- [@fluidframework/react](/packages/framework/react)</br>- [@fluidframework/request-handler](/packages/framework/request-handler)</br>- [@fluidframework/synthesize](/packages/framework/synthesize)</br>- [@fluidframework/undo-redo](/packages/framework/undo-redo) | - [Base-Definitions](#Base-Definitions)</br>- [Protocol-Definitions](#Protocol-Definitions)</br>- [Driver-Definitions](#Driver-Definitions)</br>- [Container-Definitions](#Container-Definitions)</br>- [Base-Utils](#Base-Utils)</br>- [Framework-Utils](#Framework-Utils)</br>- [Loader](#Loader)</br>- [Runtime](#Runtime)</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp; |
```

Comment:
How can I trigger this to update the fluid-framework location? I can manually change "/experimental/framework/fluid-framework" to be "/packages/framework/fluid-framework" but this file says that it would be auto-generated

Reply:
`FluidFramework/tools/build-tools$ node dist/layerCheck/layerCheck.js --root ../.. --md`

Reply:
Hmm running that gives me a number of changes that aren't related to this package specifically. Has this command not been run to update this file in some time? If so, I was thinking that we can update that in a follow-up PR to avoid combining changes for other unrelated packages like `@fluid-tools/benchmark`, `@fluidframework/server-services-telemetry`, etc in this

Reply:
Discussed in chat, we'll do a separate PR to update this file for all of the packages since it seems that this script hasn't been run in some time